onnxruntime/test/contrib_ops/function_test_util.h

```diff
@@ -23,6 +23,19 @@ inline std::vector<T> random(std::vector<int64_t> shape) {
   return generator.Uniform<T>(shape, 0.0f, 1.0f);
 }
 
+
+template <>
+inline std::vector<int64_t> random<int64_t>(std::vector<int64_t> shape) {
+  int64_t size = 1;
+  for (auto dim : shape)
+    size *= dim;
+
+  std::vector<int64_t> data(size);
+  for (int64_t i = 0; i < size; i++)
+    data[i] = static_cast<int64_t>(rand());
```

Comment:
using rand() will bypass the test random seed, consider using RandomValueGenerator instead